api/types/summarizer/summarizer.go

```diff
@@ -78,6 +78,17 @@ func ValidateInferenceModel(m *summarizerv1.InferenceModel) error {
 		if p.Openai.GetOpenaiModelId() == "" {
 			return trace.BadParameter("spec.openai.openai_model_id is required")
 		}
+	case *summarizerv1.InferenceModelSpec_Bedrock:
+		// TODO(bl-nero): Relax this condition once we implement spend controls.
+		if modules.GetModules().Features().Cloud {
+			return trace.BadParameter("Amazon Bedrock models are unavailable in Teleport Cloud")
+		}
+		if p.Bedrock.GetBedrockModelId() == "" {
+			return trace.BadParameter("spec.bedrock.bedrock_model_id is required")
```

Comment:
should we parse the model id and ensure it's a proper arn?

Reply:
(It also doesn't build my trust that this expression seems to be different in different places that accept model IDs, so it's really hard to tell which one is the one to rule them all.)